resources/content/articles/en/sh_hsk_what-is-a-stake-pool

```diff
@@ -0,0 +1,15 @@
+---
+title: What is a stake pool
```

Comment:
```suggestion
title: What is a stake pool?
```
We have a question mark in the nav heading for 'What is a Cardano node?' - so we should have one here for consistency.